### ISSUE TITLE
FoD: `fcli fod scan start-mobile` not recognising timezones

### ISSUE THREAD

**Issue by kadraman** [fortify/fcli#287]
Starting mobile scan as follows:

```
 fcli fod scan start-mobile --file .\MSTG-Android-Java.apk --entitlement SubscriptionOnly --framework Android --assessment Mobile "MASTG-Hacking-Playground:Android"
Validating entitlements...
Running scan using entitlement: 11948 - Remediation
Id       Type  Analysis Status  Application  Microservice  Release  Started  Completed  Scan Method  Action
6821401  N/A   Pending          N/A          N/A           N/A      N/A      N/A        N/A          STARTED
```

Btw, I also had to remove `--timezone` because I’m having problems to add `(UTC-03:00) Brasilia` and other quoted parameters, `--timezone UTC` and `--timezone=UTC` ran fine.

**Comment by rsenden** [comment]
Fixed by 656ca13

**Comment by kadraman** [comment]
Implemented in above commit to develop branch.
